### PR TITLE
ENT-3849 Clarify how roles affect shared reports (3.7)

### PR DESCRIPTION
**Pull request by olehermanse**
https://github.com/cfengine/documentation/pull/1824

(cherry picked from commit b6c2253a4206714f121da13c262eb6379dfc8766)

### CODE REVIEW

File: enterprise-cfengine-guide/settings.markdown [hunk 1]
```diff
@@ -49,24 +49,68 @@ users, including their name, role, and password.
...
+Roles limit access to which hosts can be seen based on the classes reported by
+the host. For example if you want to limit a users ability to report only on
+hosts in the "North American Data Center" you could setup a role that includes
+only the `location_nadc` class.
 
-If you have multiple roles assigned to a user, the user can access
-only hosts that match all of their role restrictions. For example, if
-you have the admin role and a role that matches zero hosts, the user
-will not see any hosts in Mission Portal.
+When multiple roles assigned to a user, the user can access only resources that
```
Comment (nickanderson) [inline enterprise-cfengine-guide/settings.markdown]: When multiple roles are assigned... Just missing the word are

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @craigcomstock can review this?

File: enterprise-cfengine-guide/settings.markdown (hunk 1 above)
Comment (craigcomstock) [reply enterprise-cfengine-guide/settings.markdown]: @nickanderson I made another PR as an example of what I'd like to see. I also think I fixed some other formatting inconsistencies in the example.

https://github.com/cfengine/documentation/pull/1829 (master)

**Comment by olehermanse** [comment]
@craigcomstock @nickanderson I've cherry-picked craigs additional changes as well. Please approve/merge.

**Pull request merged**